eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -45,6 +45,7 @@ library
     Cardano.Ledger.Conway.TxBody.Internal
     Cardano.Ledger.Conway.TxCert
     Cardano.Ledger.Conway.TxInfo
+    Cardano.Ledger.Conway.TxOut
```

Comment:
That is a bad idea, because `Conway does not define a custom version of `TxOut, so it makes no sense to expose this module 
```suggestion
```